cli/parser/parser.go

```diff
@@ -263,10 +263,10 @@ func formatOption(option *Option, value string) string {
 	return "--" + option.Name + "=" + value
 }
 
-// Option describes the schema for a single Bazel option.
+// OptionSchema describes the schema for a single Bazel option.
 //
 // TODO: Allow plugins to define their own option schemas.
-type Option struct {
+type OptionSchema struct {
```

Comment:
I think that Bazel uses the term `OptionDefinition`, but this is fine as well.

Reply:
imo no reason to switch away from what bazel uses; renaming takes very little time. I'll switch it over.

Reply:
Done :D